posthog/models/entity/entity.py

```diff
@@ -44,6 +45,7 @@ class Entity(PropertyMixin):
     math: Optional[MATH_TYPE]
     math_property: Optional[str]
     math_group_type_index: Optional[GroupTypeIndex]
+    math_property_type: Optional[PropertyType]
```

Comment:
Hmm, don't need this at all if special casing sessions fully, and since there's no other session property, yet.